### PR TITLE
Allow to filter in the condition list, and in the position list

### PR DESCRIPTION
**Pull request by mariano-aguero**
No issue related

Now you can filter by oracle in the condition list, and filter by collateral in the position list

### CODE REVIEW

**Comment by mariano-aguero** [comment]
Need to fix some test

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr162--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr162--conditionaltokensexplorer.review.gnosisdev.com

File: src/hooks/usePositions.tsx [hunk 1]
```diff
@@ -1,31 +1,42 @@
...
+  if (collateralFilter !== CollateralFilterOptions.All) {
+    buildQueryOptions.collateral = collateralFilter
+  }
+
+  const query = buildQueryPositions(buildQueryOptions)
+
+  const options = {
+    variables: {
+      positionId: searchPositionId,
+      collateral: collateralFilter,
```
Comment (lmcorbalan) [inline src/hooks/usePositions.tsx:33]: Shouldn't collateral only be set if  `collateralFilter !== CollateralFilterOptions.All` ?

**Review by hpmaxi — APPROVED** [review]

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr162--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr162--conditionaltokensexplorer.review.gnosisdev.com

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr162--conditionaltokensexplorer.review.gnosisdev.com

**Review by lmcorbalan — APPROVED** [review]
:racehorse:

**Comment by elena-zh** [comment]
Verified

**Pull request merged**